studio/schemas/documents/lesson.js

```diff
@@ -54,6 +55,13 @@ export default {
       description:
         'This can be used to provide a short description of the lesson.',
     },
+    {
+      name: 'repoUrl',
+      type: 'url',
+      title: 'Github Repository Url',
+      description:
+        "A link to the Github repository where the lesson's code is hosted",
+    },
```

Comment:
we already have a `repoUrl` field in here, but it could benefit from the better title and description fields